prediction_market_agent/data_models/market_data_models.py

```diff
@@ -18,8 +29,9 @@ class Market(BaseModel):
     collateralToken: HexAddress
     outcomes: list[str]
     outcomeTokenAmounts: list[OmenOutcomeToken] = []
-    outcomeTokenMarginalPrices: list[xDai] = []
+    outcomeTokenMarginalPrices: t.Optional[list[xDai]] = []
```

Comment:
If it defaults to [], it will never be None. Should the Optional be removed, or the default []?

```suggestion
    outcomeTokenMarginalPrices: t.Optional[list[xDai]]
```